package.json

```diff
@@ -142,6 +142,9 @@
       "<rootDir>/node_modules/reflux"
     ]
   },
+  "jest-junit": {
+    "output": ".artifacts/jest.junit.xml"
```

Comment:
i couldnt actually figure out why its 'jest.junit.xml' without this, but its possible we're running a version older than what their documentation shows